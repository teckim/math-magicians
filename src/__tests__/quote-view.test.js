import React from 'react';
import renderer from 'react-test-renderer';
import QuoteView from '../views/QuotesView';

describe('testing quote view', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(<QuoteView />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
