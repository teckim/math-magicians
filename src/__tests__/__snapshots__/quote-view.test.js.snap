// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing quote view should render correctly 1`] = `
<section
  className="container"
>
  <div
    className="quote"
  >
    <p
      className="quote__text"
    >
      Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
    </p>
  </div>
</section>
`;
